packages/shorebird_cli/lib/src/command.dart

```diff
@@ -117,8 +118,11 @@ abstract class ShorebirdCommand extends Command<int> {
   }
 
   Uri? get hostedUri {
-    // TODO(bryanoltman): maybe look for an environment variable instead of
-    // reading shorebird.yaml?
+    final envHostedUrl = platform.environment['SHOREBIRD_HOSTED_URL'];
```

Comment:
I think this whole getter could just be:

```dart
  Uri? get hostedUri {
    try {
      final baseUrl = platform.environment['SHOREBIRD_HOSTED_URL'] ??
          getShorebirdYaml()?.baseUrl;
      return baseUrl == null ? null : Uri.tryParse(baseUrl);
    } catch (_) {
      return null;
    }
  }
```

Reply:
yup good call, done ✅